src/components/dropdown/dropdown.tsx

```diff
@@ -142,7 +142,10 @@ export class Dropdown {
 
   disconnectedCallback() {
     this.hide();
-    this.popover.destroy();
+    
+    if (this.popper) {
```

Comment:
Popper is exposed through the popover utility but doesn't exist in this context.